### PR TITLE
adding mainnet read me

### PR DESCRIPTION
**Pull request by avkr003**

### CODE REVIEW

File: mainnet/draft_genesis.json [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+      "tally_params": null,
+      "params": {
+        "min_deposit": [
+          {
+            "denom": "uelys",
+            "amount": "100000000"
+          }
+        ],
+        "max_deposit_period": "1209600s",
+        "voting_period": "432000s",
```
Comment (cosmic-vagabond) [inline mainnet/draft_genesis.json:340]: @avkr003 this means the voting period in mainnet will last 5 days, I believe what is standard is 48 hrs

File: mainnet/draft_genesis.json [hunk 2]
```diff
@@ -0,0 +1,682 @@
...
+          {
+            "denom": "uband",
+            "amount": "30"
+          }
+        ],
+        "prepare_gas": "600000",
+        "execute_gas": "600000",
+        "client_id": "band_price_id",
+        "band_epoch": "band_epoch",
+        "price_expiry_time": "86400",
```
Comment (cosmic-vagabond) [inline mainnet/draft_genesis.json:483]: @avkr003 the price expiry time should be the shortest possible around 3-4 seconds aligned with blocktime

File: mainnet/draft_genesis.json [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+        "life_time_in_blocks": "1"
```
Comment (cosmic-vagabond) [inline mainnet/draft_genesis.json:484]: @avkr003 actually that one defined the expiry in blocks so I believe it is enough, please confirm thanks

File: mainnet/draft_genesis.json [hunk 4]
```diff
@@ -0,0 +1,682 @@
...
+          "display": "EDENB",
+          "band_ticker": "EDENB",
+          "elys_ticker": "EDENB",
+          "decimal": "6"
+        }
+      ],
+      "prices": [],
+      "price_feeders": [
+        {
+          "feeder": "elys10d07y265gmmuvt4z0w9aw880jnsr700j6z2zm3",
```
Comment (cosmic-vagabond) [inline mainnet/draft_genesis.json:513]: @avkr003 that one seems to be used in testnet as well is it intentional?

File: mainnet/draft_genesis.json [hunk 5]
```diff
@@ -0,0 +1,682 @@
...
+        "leverage_max": "25.000000000000000000",
+        "borrow_interest_rate_max": "0.300000000000000000",
+        "borrow_interest_rate_min": "0.100000000000000000",
+        "borrow_interest_rate_increase": "0.000300000000000000",
+        "borrow_interest_rate_decrease": "0.000300000000000000",
+        "health_gain_factor": "0.000000220000000000",
+        "max_open_positions": "100000",
+        "pool_open_threshold": "0.650000000000000000",
+        "borrow_interest_payment_fund_percentage": "0.100000000000000000",
+        "borrow_interest_payment_fund_address": "elys1xd986tqfwwcxeywcmsvq62mwk4z8cvzsfnplt3",
```
Comment (cosmic-vagabond) [inline mainnet/draft_genesis.json:547]: @avkr003 that address seems to be the same as in testnet

File: mainnet/draft_genesis.json [hunk 6]
```diff
@@ -0,0 +1,682 @@
...
+    },
+    "tier": {
+      "params": {},
+      "portfolio_list": []
+    },
+    "tokenomics": {
+      "params": {},
+      "airdrop_list": [],
+      "genesis_inflation": null,
+      "time_based_inflation_list": []
```
Comment (cosmic-vagabond) [inline mainnet/draft_genesis.json:620]: @avkr003 do we need to define any tokenomics here?

File: mainnet/draft_genesis.json [hunk 7]
```diff
@@ -0,0 +1,682 @@
...
+        "pool_creation_fee": "10000000",
+        "slippage_track_duration": "604800",
+        "base_assets": [],
+        "weight_breaking_fee_exponent": "2.500000000000000000",
+        "weight_breaking_fee_multiplier": "0.000500000000000000",
+        "weight_breaking_fee_portion": "0.500000000000000000",
+        "weight_recovery_fee_portion": "0.100000000000000000",
+        "threshold_weight_difference": "0.050000000000000000",
+        "allowed_pool_creators": [
+          "elys10d07y265gmmuvt4z0w9aw880jnsr700j6z2zm3",
```
[the remark this answers is not in the document]
Comment (avkr003) [reply mainnet/draft_genesis.json:21]: yes. It gov module address

File: mainnet/draft_genesis.json [hunk 8]
```diff
@@ -0,0 +1,682 @@
...
+        "consumer_redistribution_fraction": "0.75",
+        "historical_entries": "10000",
+        "unbonding_period": "1728000s",
+        "soft_opt_out_threshold": "0",
+        "reward_denoms": [],
+        "provider_reward_denoms": [],
+        "retry_delay_period": "3600s",
+        "consumer_id": "0"
+      },
+      "provider": {
```
[the remark this answers is not in the document]
Comment (avkr003) [reply mainnet/draft_genesis.json:122]: yup. to generate this we need to stop opt in

File: mainnet/draft_genesis.json (hunk 4 above)
Comment (avkr003) [reply mainnet/draft_genesis.json:513]: gov address

File: mainnet/draft_genesis.json (hunk 3 above)
Comment (avkr003) [reply mainnet/draft_genesis.json:484]: I still have to validate this. Putting in draft.

File: mainnet/draft_genesis.json (hunk 6 above)
Comment (avkr003) [reply mainnet/draft_genesis.json:620]: Post launch through proposal

File: mainnet/draft_genesis.json (hunk 1 above)
Comment (avkr003) [reply mainnet/draft_genesis.json:340]: Standard is 5 days
https://www.mintscan.io/osmosis/parameters
https://www.mintscan.io/cosmos/parameters

File: mainnet/draft_genesis.json (hunk 5 above)
Comment (avkr003) [reply mainnet/draft_genesis.json:547]: its module address

File: mainnet/draft_genesis.json (hunk 2 above)
Comment (avkr003) [reply mainnet/draft_genesis.json:483]: Have to update it. Not final version. It's draft

File: mainnet/draft_genesis.json (hunk 7 above)
[the remark this answers is not in the document]
Comment (cosmic-vagabond) [reply mainnet/draft_genesis.json:21]: oh got it thanks for clarifying

**Review by cosmic-vagabond — APPROVED** [review]

**Pull request merged**